README.md

```
@@ -2,61 +1,32 @@
-Let us know your thoughts below. And good luck with development!
+---
 
 ## Table of Contents
 
 - [Versions](#versions)
-- [Demo](#demo)
-- [Quick Start](#quick-start)
+- [Terminal Commands](#terminal-commands)
 - [Deploy](#deploy)
 - [Documentation](#documentation)
 - [File Structure](#file-structure)
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)